docker-compose.yml

```diff
@@ -1,11 +1,30 @@
-version: '2'
+version: '3.7'
 
 services:
-  db:
-    image: postgres:11
+  {{name}}:
+    image: {{name}}:latest
```

Comment:
Should this be `vapor/swift:5.1`?

Reply:
Not the way I’ve got it set up. I’ve got it set up to build web.Dockerfile and then name the result {{name}}:latest

So {{name}}:latest refers to the custom image representing your app, not the base image that the Dockerfile pulls in at the top.

Reply:
Ah okay, that makes sense. Thanks for explaining.